src/relay/qnn/op/requantize.cc

```diff
@@ -263,6 +263,14 @@ bool RequantizeRel(const Array<Type>& types, int num_inputs, const Attrs& attrs,
     return false;
   }
 
+  if (types[0].as<IncompleteTypeNode>()) {
```

Comment:
@mbrookhart I thin this is not necessary, as this case is already covered by https://github.com/apache/tvm/blob/main/src/relay/qnn/op/requantize.cc#L260-L264 above

Reply:
Good catch, thanks!